### DISCUSSION TITLE
Looking for the STL for case

### DISCUSSION THREAD

**Discussion by milesmckinnon** [revk/xESP32-Faikout#176] [General]
Hi there,

Thanks @revk for all your work on this project! I've just ordered my boards and can't wait to get this setup.

I'm looking for the STL file to print the case. I found the Open SCAD program which looks like it generates the STL from the KiCAD design. I'm not very skilled in Open SCAD, and was wondering if the STL could be posted to the file library?

Thanks again!

**Comment by revk** [comment]
The PCB/Faikin/Faikin.stl

**Reply by milesmckinnon** [reply]
Thank you!
